incubator/kiam/templates/server-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.rbac.create -}}
```

Comment:
Add `.Values.server.enabled` to the condition.

Reply:
👍